modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/filters/FiltersFragment.kt

```diff
@@ -183,6 +214,70 @@ class FiltersFragment :
         playlistFragment.view?.requestFocus() // Jump to new page for talk back
     }
 
+    private fun showTooltip() {
+        val toolbar = binding?.toolbar ?: return
+        binding?.tooltipComposeView?.isVisible = true
+        binding?.tooltipComposeView?.apply {
+            setContentWithViewCompositionStrategy {
+                AppTheme(theme.activeTheme) {
+                    val configuration = LocalConfiguration.current
+                    var toolbarY by remember { mutableFloatStateOf(0f) }
+
+                    CallOnce {
+                        viewModel.trackTooltipShown()
+                    }
+
+                    LaunchedEffect(configuration) {
+                        val location = IntArray(2)
+                        toolbar.getLocationOnScreen(location)
+                        toolbarY = (location[1] + toolbar.height).toFloat()
+                    }
+
+                    Box(
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .background(Color.Black.copy(alpha = 0.4f))
+                            .clickable(
+                                interactionSource = null,
+                                indication = null,
+                                onClick = { closeTooltip() },
+                            )
+                            .semantics {
+                                invisibleToUser()
+                            },
+                    ) {
+                        Layout(
+                            content = {
+                                FiltersTooltip(
+                                    onClickClose = { closeTooltip() },
+                                    modifier = Modifier.widthIn(max = 326.dp).padding(horizontal = 16.dp),
```

Comment:
| Design | Implementation |
| --- | --- |
| <img width="698" alt="Screenshot 2025-03-27 at 3 24 56 pm" src="https://github.com/user-attachments/assets/bdcec5e7-040e-4fd9-9df5-960ebe617b39" /> | <img width="698" alt="Screenshot 2025-03-27 at 3 24 56 pm" src="https://github.com/user-attachments/assets/94e61424-76be-4d17-8f06-ed09497fcf30" /> | 

I'm not sure if it's only on my test device but is it possible to line up the arrow closer to the plus icon? Should the description text also be slightly smaller?

Reply:
It's resolved now!